__integrations__/index.test.ts

```diff
@@ -406,6 +408,33 @@ describe("Create Message", () => {
       expect(resultGet.status).toEqual(200);
       const detail = (await resultGet.json()) as ExternalMessageResponseWithContent;
 
+      await pipe(
+        {
+          message: messageModel.find([messageId as NonEmptyString, fiscalCode]),
+          status: messageStatusModel.findLastVersionByModelId([
+            messageId as NonEmptyString
+          ])
+        },
+        sequenceS(TE.ApplicativePar),
+        TE.bindW("content", _ =>
+          pipe(
+            messageModel.getContentFromBlob(
+              blobService,
+              messageId as NonEmptyString
+            ),
+            TE.orElseW(_ => TE.of(O.none as O.Option<MessageContent>))
+          )
+        ),
+        TE.mapLeft(_ => fail(`Error retrieving message data from Cosmos.`)),
+        TE.map(({ message, status, content }) => {
+          expect(O.isSome(message)).toBeTruthy();
+          expect(O.isSome(status)).toBeTruthy();
+          expect(O.isSome(content)).toBeFalsy();
+          expect(O.some(status)).not.toHaveProperty("ttl");
+          expect(O.some(message)).not.toHaveProperty("ttl");
```

Comment:
`O.some` of an Option<Message> is  Option<Option<Message>>. 
We need to extract the value of the option to check the existance of the `ttl` property.